config/initializers/rack/attack.rb

```diff
@@ -16,4 +16,10 @@ class Rack::Attack
       request.env["HTTP_API_KEY"]
     end
   end
+
+  throttle("site_hits", limit: 20, period: 1) do |request|
```

Comment:
In thinking about this now, even 20 in one is not enough for how we handle at least one poorly-handled situation:

Open up the network tab and go here: https://dev.to/search?q=ben&filters=class_name:User

I think it's worth refactoring this situation to handle a bulk following request that handles _all_ the follow buttons on the page at once for whether there is a follow like how we do reactions on comment thread pages....

But for now it would cause problems here.

Either we merge this now as something like `limit: 75, period: 1)`

Or even `limit: 100, period: 3)` or something like that...

But we could hold off and try to adjust the way we bulk-fetch follow checks? I could probably whip up the long-overdue change.

Reply:
lol oh that's a good one, about 100 requests in 2 seconds. Though a bunch of them look like they are being made to cloudinary and not dev. I like 100 for a period of 3 and I [made a ticket to optimize that situation](https://github.com/thepracticaldev/dev.to/projects/9#card-36804382)